faucet/Cargo.toml

```diff
@@ -15,11 +15,11 @@ actix-files = "0.6.5"
 actix-cors = "0.7.0"
 derive_more = "0.99.17"
 figment = { version = "0.10", features = ["toml", "env"] }
-miden-lib = { workspace = true }
+miden-lib = { version = "0.1.0" }
 miden-client = { version = "0.1.0", features = ["concurrent"] }
 miden-node-proto = { path = "../proto", version = "0.2" }
 miden-node-utils = { path = "../utils", version = "0.2" }
-miden-objects = { workspace = true }
+miden-objects = { version = "0.1.0" }
 serde = { version = "1.0", features = ["derive"] }
```

Comment:
Why are these changes needed?

Reply:
@bobbinth I faced incompatibility of miden-client with the `next` branch of miden-base and decided not to migrate it yet, just use previous version in the faucet. Today @igamigo contacted me and provided updated code, so I will check and probably update the deps.

Reply:
@bobbinth update: migration of the fauced seems not a straightforward, needs to rewrite part of code regarding using of new miden-client. For this PR I leave old version of the client.
@igamigo @phklive FYI

Reply:
@phklive some of structures/arguments were changed in miden-client, thus, miden-faucet code needs to be also updated.

Reply:
Understood. We will soon rework the faucet + end-to-end testing code to remove the dependency on the client. We will implement transaction creation, execution and submission directly using the base, making sure that we do not have these types of problems in the future.

Reply:
Didn't the faucet use the released client version? There shouldn't have been any changes in that regard so I wonder  what the issue was. One thing that will break on the node when bringing everything to `next` is the account generation code which still uses `KeyPair`. I will take a look.

Reply:
```rust
miden-client = { version = "0.1.0", features = ["concurrent"] }
miden-node-proto = { path = "../proto", version = "0.2" }
miden-node-utils = { path = "../utils", version = "0.2" }
```
Seems like the faucet should probably use the release versions of `miden-node-proto` in order to be fully compatible with the client. https://github.com/0xPolygonMiden/miden-node/blob/next/faucet/Cargo.toml